manifests/config.pp

```diff
@@ -59,6 +63,14 @@
       onlyif  => "test `authselect current | grep -i profile | grep -i ${authselect_profile} | wc -l` -lt 1",
       command => "authselect select ${authselect_profile} --force",
     }
+    # FOR ITEMS IN $authselect_features ... $authselect_feature
+    $authselect_features.each | $authselect_feature | {
+      exec { "authselect_enable_feature_${authselect_feature}":
+        path    => '/sbin/:/bin/:/usr/bin/:/usr/sbin/',
+        onlyif  => "test `authselect current | grep -i -A20 features | grep -iv features | grep -i ${authselect_feature} | wc -l` -lt 1",
+        command => "(authselect enable-feature ${authselect_feature}) || (authselect select ${authselect_profile} --force && authselect enable-feature ${authselect_feature})",
+      }
+    }
```

Comment:
This should be changed to one single exec resource that joins all of the authselect features into a single string, then runs something like `authselect select sssd with-faillock with-mkhomedir --force`. Of course the `onlyif` parameters will also need to be adjusted to check for the authselect profile and each authselect feature as well.